imagetool/src/main/resources/docker-files/Create_Image.mustache

```diff
@@ -190,8 +196,11 @@ ARG ADMIN_NAME
 ARG ADMIN_HOST
 ARG ADMIN_PORT
 ARG MANAGED_SERVER_PORT
+ARG ORAINST
```

Comment:
Why do we need this argument and environment variable? Looks like they could be removed.

Reply:
Fixed the requested changes